pyproject.toml

```diff
@@ -2,3 +2,16 @@
 addopts = [
     "--import-mode=importlib",
 ]
+
+
+[tool.flake8.messages_control]
+disable = [
+    "--max-line-length",
+]
+
+[tool.flake8]
+--max-line-length = 120
+
+[flake8]
+max-line-length = 100
```

Comment:
I will experiment shortly to confirm which actual solution worked to solve our issue.